### DISCUSSION TITLE
Spatie for SPA with authorization gates

### DISCUSSION THREAD

**Discussion by bashiirmahad** [spatie/laravel-permission#2400] [General]
I want to create a Laravel application for SPA utilizing Spatie roles & permissions,  using gates in the blade is simple for me. However, API is a bit complicated, so please provide any suggestions for the best approach to accomplish this. thanks

**Comment by drbyte** [comment]
It's not clear to me what you're asking.

**Comment by parallels999** [comment]
> so please provide any suggestions for the best approach to accomplish this

watch tutorials on the internet
https://youtu.be/ZWUCQMoKW4g
https://youtu.be/ncrZ3yQi3nc

**Reply by bashiirmahad** [reply]
Thank you very much
